### PR TITLE
Add playwright test for unsupported browser modal

### PR DESCRIPTION
**Pull request by timoballard**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/823)

## What does this change?
Adds a playwright test to assert the visibility of the unsupported browser modal when the user agent string indicates that the Samsung browser is in use. 

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/tests/integration/unsupported_browser.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+@pytest.fixture(scope="function")
+def browser_context_args(browser_context_args):
+    return {
+        **browser_context_args,
+        "userAgent": "SamsungBrowser/i"
```
Comment (cantsin) [inline crt_portal/cts_forms/tests/integration/unsupported_browser.py:8]: `/i` (case-insensitive regexp flag) is not necessary here, can we remove?

**Review by cantsin — CHANGES_REQUESTED** [review]
:star_struck: LGTM, just curious about the `/i` bit

File: crt_portal/cts_forms/tests/integration/unsupported_browser.py (hunk 1 above)
Comment (timoballard) [reply crt_portal/cts_forms/tests/integration/unsupported_browser.py]: ... caught my copy/pasting 😂 

✔️

**Review by cantsin — APPROVED** [review]
LGTM x2!

**Pull request merged**